r/R/filesystem.R

```diff
@@ -442,6 +442,57 @@ default_s3_options <- list(
   request_timeout = -1
 )
 
+# Enumerate possible values for AWS SDK log levels, used by s3_init
+s3_log_levels <- c("off", "fatal", "error", "warn", "info", "debug", "trace")
```

Comment:
Instead of defining these here, you can set the `log_level` parameter in the function definition to this vector instead.

Reply:
Good point, seems fine to just do that.